PersonaStats.Rmd

```diff
@@ -0,0 +1,287 @@
+---
+title: "Personas"
+author: "Kate Isaac"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r}
+library(tidyverse)
+library(dplyr)
+library(data.table)
+library(magrittr)
+library(here)
+```
+
+```{r}
+resultsTidy <- readRDS(here("data/resultsTidy.rds"))
+```
+
+```{r}
+kindOfWorkDF <- resultsTidy %>%
+  separate(KindOfWork,
+           c("whichWorkA", "whichWorkB", "whichWorkC", "whichWorkD", "whichWorkE", "whichWorkF", "whichWorkG", "whichWorkH", "whichWorkI", "whichWorkJ"),
+           sep=", ", fill="right") %>%
+  pivot_longer(starts_with("whichWork"), 
+               values_to = "whichWorkDescription") %>%
+  dplyr::select(Timestamp, UserType, whichWorkDescription) %>%
+  mutate(whichWorkDescription =
+           recode(whichWorkDescription,
+                  "computational education" = "Computational education",
+                  "Program administration," = "Program administration"),
+         whichWorkDescription =
+           factor(whichWorkDescription),
+         Timestamp = 
+           factor(Timestamp)
+         ) %>%
+  drop_na() %>% 
+  pivot_wider(id_cols = Timestamp,
+              names_from = whichWorkDescription,
+              values_from = whichWorkDescription)
```

Comment:
Could explore this as a more concise alternative:
```
resultsTidy %>% 
  select(Timestamp, KindOfWork) %>% 
  separate_longer_delim(KindOfWork, delim = ", ") %>% 
  mutate(value = TRUE) %>% 
  pivot_wider(names_from = KindOfWork, values_from = value, values_fill = FALSE)
```

Reply:
That's a really cool alternative!